phala-cloud/confidential-ai/confidential-gpu/overview.mdx

```diff
@@ -0,0 +1,44 @@
+---
+description: Rent Confidential GPU on Phala Cloud with various GPU device and flexible subscription.
+title: Confidential GPU
+---
+
+## Overview
+
+Rent a Phala Cloud GPU Server with Confidential Computing capabilities. Deploy your models (training or inference) in a TEE-secured environment, just like a standard GPU server.
+
+## Available GPU Options
+
+Choose from high-performance GPUs:
+
+- H200
+- H100
+- B200
+
+<Frame caption="GPU Device">
+    <img src="/images/confidential-ai/confidential-gpu/gpu-tee-01.png" />
+</Frame>
+
+## Customizable Hardware
+
+Select your preferred configuration:
+
+- CPU, RAM, Storage
+- Location (data center regions)
+- GPU Quantity (1 GPU or up to 8 GPUs for scaling)
+
+## Commitment Period
+
+Flexible subscription options:
+
+- 6 months
+- 1 month
+- On-demand (pay-as-you-go)
+
+## Review & Order
```

Comment:
We should mention what will happen after submitting the order.

It's not really "completing your purchase". Instead, it send an order to the team, and we will contact the user to follow up.

Then we should say once the GPU is provisioned, the user can attach GPU TEE to their CVM. It allows users to deploy arbitrary containerized app to GPU TEE.